release-notes/10.0/preview/preview1/10.0.0-preview.1.md

```diff
@@ -0,0 +1,67 @@
+# .NET 10 Preview 1 Binaries
+
+.NET 10 Preview 1 installers and binaries can be downloaded for the following product versions:
+
+- `.NET 10.0.0 Preview 1`
+- `.NET SDK 10.0.100-preview.1.XXXXX.X`
+
+See [README.md](README.md) for .NET 10 release notes and discussions.
+
+The latest .NET releases are always listed at [.NET Releases](../../README.md).
+
+## Downloads
+
+// To add download links
+
+1. Includes the .NET Runtime and ASP.NET Core Runtime
+2. For hosting stand-alone apps on Windows Servers. Includes the ASP.NET Core Module for IIS and can be installed separately on servers without installing .NET Runtime.
+
+The .NET SDK includes a matching updated .NET Runtime. Downloading the Runtime or ASP.NET Core packages is not needed when installing the SDK.
+
+You can check your .NET SDK version by running the following command. The example version shown is for this release.
+
+```console
+$ dotnet --version
+10.0.100-preview.1.XXXXX.X
+```
+
+## .NET Multi-Platform App UI (MAUI) Workload Downloads
+
+ After installing the .NET SDK, you can install .NET MAUI using the `dotnet workload install` command:
+
+ ```console
+ dotnet workload install maui
+ ```
+
+ The following workloads are also available to install individually:
+
+ ```console
+ dotnet workload install android
+ dotnet workload install ios
+ dotnet workload install maccatalyst
+ dotnet workload install macos
+ dotnet workload install tvos
+ ```
+
+## Container Images
+
+[.NET container images](https://github.com/dotnet/dotnet-docker/blob/main/README.md) have been updated for this release. The [.NET container samples](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md) show various ways to use .NET container images.
+
+The following repos have been updated:
+
+- [dotnet/sdk](https://github.com/dotnet/dotnet-docker/blob/main/README.sdk.md): .NET SDK
+- [dotnet/aspnet](https://github.com/dotnet/dotnet-docker/blob/main/README.aspnet.md): ASP.NET Core Runtime
+- [dotnet/runtime](https://github.com/dotnet/dotnet-docker/blob/main/README.runtime.md): .NET Runtime
+- [dotnet/runtime-deps](https://github.com/dotnet/dotnet-docker/blob/main/README.runtime.md): .NET Runtime Dependencies

```

Comment:
@richlander @MichaelSimons thoughts on adding `docker pull` commands inline here?

Reply:
Can you illustrate what you are thinking (e.g. for just one repo)?

Reply:
```suggestion
- [dotnet/sdk](https://github.com/dotnet/dotnet-docker/blob/main/README.sdk.md)
  - `docker pull mcr.microsoft.com/dotnet/sdk:10.0-preview`
- [dotnet/aspnet](https://github.com/dotnet/dotnet-docker/blob/main/README.aspnet.md)
  - `docker pull mcr.microsoft.com/dotnet/aspnet:10.0-preview`
- [dotnet/runtime](https://github.com/dotnet/dotnet-docker/blob/main/README.runtime.md)
  - `docker pull mcr.microsoft.com/dotnet/runtime:10.0-preview`
- [dotnet/runtime-deps](https://github.com/dotnet/dotnet-docker/blob/main/README.runtime.md)
  - `docker pull mcr.microsoft.com/dotnet/runtime-deps:10.0-preview`
```

Reply:
@MichaelSimons good with this?